### DISCUSSION TITLE
Installation for windows the way it's implemented in automatic1111

### DISCUSSION THREAD

**Discussion by mockinbirdy** [oobabooga/text-generation-webui#3132] [Ideas]
Hi, 

May someone create an installation file for windows the way it's implemented in automatic1111? Since this repo is referring to it as the inspiration, it would be pretty cool to have a **simple** way to install it, with `venv` inside the folder that can be easily deleted and quickly reinstalled when something goes wrong. There is no need to have a separate anaconda folder inside which actually messes things up more than helps. The incompatibility of components is nightmarish. 

It significantly slows down the adoption of this webui by ordinary users, and that is sad. Hope someone who actually uses windows will look into this and fix this major issue. Would be really, really cool.

**Comment by BarfingLemurs** [comment]
@mockinbirdy yea but this wont work, you need to install alot of things that aren't "pip installable" hence conda

**Reply by BarfingLemurs** [reply]
https://github.com/oobabooga/text-generation-webui/discussions/1566 here's another way you can enter your enviorment
